frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/UnmanagedBranchNotificationFactory.java

```diff
@@ -127,9 +127,7 @@ private NotificationAction getOpenMacheteFileAction() {
           val dataContext = new DataContext() {
             @Override
             public @Nullable Object getData(String dataId) {
-              return Match(dataId).of(
-                  typeSafeCase(CommonDataKeys.PROJECT, project),
-                  Case($(), (Object) null));
+              return dataId.equals(CommonDataKeys.PROJECT.getName()) ? project : null;
```

Comment:
why don't we use `switch` here too? I know, logically it's a bit redundant but imho it's more readable 

and what about the other `DataContext#getData`s we define? Why don't we have switch there? 🤔

Reply:
`switch` only accepts constant expressions for `case` ☹️ so Vavr `Match` is still left in a few places